lib/locales/en.ts

```diff
@@ -925,10 +948,12 @@ export const locale: Locale = {
     teresa_of_jesus_of_avila_virgin: 'Saint Teresa of Jesus, Virgin and Doctor of the Church',
     teresa_of_jesus_of_los_andes_virgin: 'Saint Teresa of Jesus of Los Andes, Virgin',
     teresa_of_portugal_religious: 'Blessed Teresa of Portugal, Religious',
+    the_penitent_thief: 'The Penitent Thief',
     theodore_of_canterbury_bishop: 'Saint Theodore of Canterbury, Bishop',
     theodore_romzha_bishop: 'Blessed Theodore Romzha, Bishop and Martyr',
     theodosius_of_the_caves_abbot: 'Saint Theodosius of the Caves, Abbot',
     theotonius_of_coimbra_priest: 'Saint Theotonius of Coimbra, Priest',
+    therese_couderc_virgin: 'Saint Thérèse Couderc, Virgin',
```

Comment:
Add first name to the key name.

Is she really celebrated as _virgin_? According to [Wikipedia](https://en.wikipedia.org/wiki/Th%C3%A9r%C3%A8se_Couderc), she was a co-founder of Sisters of the Cenacle, therefore she could be considered a religious.

```suggestion
    therese_marie_victoire_couderc_virgin: 'Saint Thérèse Couderc, Virgin',
```